packages/primitives/types/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Definitions by:
+// Junyoung Clare Jang <https://github.com/Ailrun>
+// Sam Pettersson <https://github.com/iamsamwhoami>
+// TypeScript Version: 2.3
+
+import {
+  CreateStyled,
+  Interpolation,
+  StyledComponent,
+  StyledOptions,
+  Themed,
+  CreateStyledOtherComponent
+} from '@emotion/primitives-core';
+
+import { BaseInterpolation } from '@emotion/primitives-core/types/common';
+
+import { ImageProps, ViewProps, TextProps } from 'react-native';
+
+export * from 'emotion';
+
+export type ThemedReactEmotionInterface<Theme extends object> = CreateStyled<
+  Theme
+>;
+
+declare module 'react-native' {
+  interface ViewProps {
```

Comment:
Typings for `react-primitives` doesn't seem to expose these, so taking them from `react-native` typings instead.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-primitives/index.d.ts